backend/src/index.js

```diff
@@ -23,12 +24,17 @@ app.get('/', (req, res) => {
 app.get(
   '/candidates',
   errorWrap(async (req, res) => {
-    try {
-      const candidates = await Candidate.find()
-      res.json({ result: candidates })
-    } catch (err) {
-      res.json(400, { message: err.message })
+    let candidates
+    if (req.query.status) {
+      if (req.query.status == 'everyone') {
```

Comment:
this is a little redundant since it returns the same thing if you don't include `?status`. but if you have a use case for it, that's fine. Just watch out for this stuff

Reply:
and resolve your merge conflicts